packages/headless/src/controllers/insight/facets/range-facet/numeric-facet/headless-insight-numeric-facet.ts

```diff
@@ -0,0 +1,101 @@
+import {NumericRangeRequest} from '../../../../../features/facets/range-facets/numeric-facet-set/interfaces/request';
+import {NumericFacetValue} from '../../../../../features/facets/range-facets/numeric-facet-set/interfaces/response';
+import {
+  ConfigurationSection,
+  NumericFacetSection,
+  SearchSection,
+} from '../../../../../state/state-sections';
+import {
+  configuration,
+  numericFacetSet,
+  search,
+} from '../../../../../app/reducers';
+import {loadReducerError} from '../../../../../utils/errors';
+import {getAnalyticsActionForToggleRangeFacetSelect} from '../../../../../features/facets/range-facets/generic/range-facet-utils';
+import {
+  buildCoreNumericFacet,
+  buildNumericRange,
+  NumericFacet,
+  NumericFacetOptions,
+  NumericFacetProps,
+  NumericFacetState,
+  NumericRangeOptions,
+} from '../../../../core/facets/range-facet/numeric-facet/headless-core-numeric-facet';
+import {RangeFacetSortCriterion} from '../../../../../features/facets/range-facets/generic/interfaces/request';
+import {InsightEngine} from '../../../../../app/insight-engine/insight-engine';
+import {executeSearch} from '../../../../../features/insight-search/insight-search-actions';
+import {
+  logFacetClearAll,
+  logFacetUpdateSort,
+} from '../../../../../features/facets/facet-set/facet-set-insight-analytics-actions';
+
+export type {
+  NumericRangeOptions,
+  NumericRangeRequest,
+  NumericFacetValue,
+  NumericFacetOptions,
+  NumericFacetProps,
+  NumericFacet,
+  NumericFacetState,
+};
+export {buildNumericRange};
+
+/**
+ * Creates a `NumericFacet` controller instance.
+ *
+ * @param engine - The headless engine.
+ * @param props - The configurable `NumericFacet` properties.
+ * @returns A `NumericFacet` controller instance.
```

Comment:
```suggestion
 * @returns An `InsightNumericFacet` controller instance.
```

Reply:
I'll keep `NumericFacet` to match the function signature.